proton-c/bindings/cpp/include/proton/message.hpp

```diff
@@ -329,11 +329,7 @@ class message {
     mutable annotation_map message_annotations_;
     mutable annotation_map delivery_annotations_;
 
-    /// Decode the message corresponding to a delivery from a link.
```

Comment:
This is a private function that didn't need exposing. So I moved it to the internals.

I can't remember anymore if this change was enabled by other things changed or not. But the code moved into message_adapter.cpp which also changed a lot in the same change.